readthedocs/core/templatetags/privacy_tags.py

```diff
@@ -18,9 +20,13 @@ def is_admin(user, project):
 
 @register.simple_tag(takes_context=True)
 def get_public_projects(context, user):
+    builds = Build.objects.filter(project=OuterRef('pk'))
```

Comment:
Can you add some comments here describing the `subquery` and others and why have you gone this approach?

Reply:
okay